crates/contexts/src/keystore.rs

```diff
@@ -9,7 +9,6 @@ pub trait KeystoreContext {
 
 impl KeystoreContext for GadgetConfiguration {
     fn keystore(&self) -> Keystore {
-        // TODO(XXX): This needs to be the keystore URI, not the data dir. Completely different responsibilities
```

Comment:
I think this is done cause I see keystore URI used, can you confirm @Serial-ATA @Tjemmmic ?